### ISSUE TITLE
Unblocking non-https traffic on port 443

### ISSUE THREAD

**Issue by saadusman17** [mitmproxy/mitmproxy#667]
Hi, 

If there is an app which generates some non-https traffic on port 443, the mitm proxy in transparent mode doesn't let it through, rather it answers with:

HTTP/1.1 400 Bad Request
Server: mitmproxy 0.12.1
Content-type: text/html
Content-Length: 387
Connection: close

            <html>
                <head>
                    <title>400 Bad Request</title>
                </head>
                <body>SSL handshake error: The client may not trust the proxy's certificate.<pre>ProxyError('NetLibError("SSL handshake error: Error([(\'SSL routines\', \'SSL23_GET_CLIENT_HELLO\', \'unknown protocol\')],)",)',)</pre></body>
            </html>
        
is there some configuration where I can configure that if there is an "SSL handshake error", then mitm should just let this traffic through without trying man in the middle and just pass the response back to the client machine (also without mitm). And later if a valid another session comes on port 443 which doesn't have "SSL handshake error" it should attempt man in the middle on it. 

If this option is not available as a feature, can you please give me some guidelines on which part of the code should I modify to achieve this?

Thanks,
Saad

**Comment by mhils** [comment]
Hi there,

If there's a distinctive destination host, you probably can just just `--ignore` (not logged at all then) if that fits for you. Apart from this, this issue seems to be a duplicate of #483.

Cheers,
Max
